roles/kubernetes/secrets/templates/openssl.conf.j2

```diff
@@ -26,3 +26,9 @@ IP.{{ 2 * loop.index }} = {{ hostvars[host]['ip'] | default(hostvars[host]['ansi
 {% set idx =  groups['kube-master'] | length | int * 2 + 1 %}
 IP.{{ idx }} = {{ kube_apiserver_ip }}
 IP.{{ idx + 1 }} = 127.0.0.1
+{% if supplementary_addresses_in_ssl_keys is defined %}
+{% set is = idx + 1 %}
```

Comment:
Shouldn't this be `idx + 2`?

Reply:
I was using +2 in the beginning but there was a shift. Now I use +1.

Sample of openssl.conf with +1:

```
[req]
req_extensions = v3_req
distinguished_name = req_distinguished_name
[req_distinguished_name]
[ v3_req ]
basicConstraints = CA:FALSE
keyUsage = nonRepudiation, digitalSignature, keyEncipherment
subjectAltName = @alt_names
[alt_names]
DNS.1 = kubernetes
DNS.2 = kubernetes.default
DNS.3 = kubernetes.default.svc
DNS.4 = kubernetes.default.svc.cluster.local
DNS.5 = localhost
DNS.6 = 10.0.4.110
DNS.7 = 10.0.4.111
DNS.8 = 10.0.4.112
IP.1 = 10.0.4.110
IP.2 = 10.0.4.110
IP.3 = 10.0.4.111
IP.4 = 10.0.4.111
IP.5 = 10.0.4.112
IP.6 = 10.0.4.112
IP.7 = 10.233.0.1
IP.8 = 127.0.0.1
IP.9 = 10.0.0.1      # supplementary adresses begin here
IP.10 = 10.0.0.2
IP.11 = 10.0.0.3

```